t/t7900-maintenance.sh

```diff
@@ -578,6 +578,23 @@ test_expect_success 'start and stop macOS maintenance' '
 	test_line_count = 0 actual
 '
 
+test_expect_success 'use launchctl list to prevent extra work' '
+	# ensure we are registered
+	GIT_TEST_MAINT_SCHEDULER=launchctl:./print-args git maintenance start &&
+
+	# do it again on a fresh args file
+	rm -f args &&
```

Comment:
Maybe we should also include `"$HOME/Library/LaunchAgents"` in that `rm -f` call?

Reply:
My mistake, that is just a directory, and while we list its contents into `actual`, we do not really do anything with it, it's just for debugging, I assume.